charts/csi-secrets-store-provider-azure/README.md

```diff
@@ -6,32 +6,14 @@ Azure Key Vault provider for Secrets Store CSI driver allows you to get secret c
 
 | Helm Chart Version | Secrets Store CSI Driver Version | Azure Key Vault Provider Version |
 | ------------------ | -------------------------------- | -------------------------------- |
-| `0.0.5`            | `0.0.9`                          | `0.0.5`                          |
-| `0.0.6`            | `0.0.10`                         | `0.0.5`                          |
-| `0.0.7`            | `0.0.11`                         | `0.0.6`                          |
-| `0.0.8`            | `0.0.11`                         | `0.0.7`                          |
-| `0.0.9`            | `0.0.12`                         | `0.0.7`                          |
-| `0.0.10`           | `0.0.13`                         | `0.0.8`                          |
-| `0.0.11`           | `0.0.14`                         | `0.0.9`                          |
-| `0.0.12`           | `0.0.15`                         | `0.0.9`                          |
-| `0.0.13`           | `0.0.16`                         | `0.0.9`                          |
-| `0.0.14`           | `0.0.17`                         | `0.0.10`                         |
-| `0.0.15`           | `0.0.18`                         | `0.0.11`                         |
-| `0.0.16`           | `0.0.19`                         | `0.0.12`                         |
-| `0.0.17`           | `0.0.20`                         | `0.0.13`                         |
-| `0.0.18`           | `0.0.21`                         | `0.0.14`                         |
-| `0.0.19`           | `0.0.22`                         | `0.0.15`                         |
-| `0.0.20`           | `0.0.23`                         | `0.0.16`                         |
-| `0.1.0`            | `0.1.0`                          | `0.1.0`                          |
-| `0.2.0`            | `0.2.0`                          | `0.2.0`                          |
-| `0.2.1`            | `0.3.0`                          | `0.2.0`                          |
```

Comment:
This list is getting longer, so removing the versions for old unsupported releases